include/mbedtls/cmac.h

```diff
@@ -67,16 +69,15 @@ struct mbedtls_cmac_context_t
  *                      Must be called with an initialized cipher context.
  *
  * \param ctx           The cipher context used for the CMAC operation, initialized
- *                      as one of the following types:<ul>
- *                      <li>MBEDTLS_CIPHER_AES_128_ECB</li>
- *                      <li>MBEDTLS_CIPHER_AES_192_ECB</li>
- *                      <li>MBEDTLS_CIPHER_AES_256_ECB</li>
- *                      <li>MBEDTLS_CIPHER_DES_EDE3_ECB</li></ul>
+ *                      as one of the following types: MBEDTLS_CIPHER_AES_128_ECB,
```

Comment:
I'm afraid that this will be less readable than before...

Reply:
I agree, but we discovered an issue with lists and paragrpah breaks in parameter/member documentation in RTF output (completely break tables), which is why bullets were reverted in them. Hopefully future versions of Doxygen will fix the issue, and then we will restore this.